pyhap/accessory.py

```diff
@@ -74,6 +74,18 @@ def _set_services(self):
         """
         pass
 
+    @property
+    def available(self):
```

Comment:
This makes the `self.reachable = True` assignment in init obsolete.

Reply:
I think its can simply go away since its not used anywhere in the code AFAICT